### DISCUSSION TITLE
v1.2.0 - Outposts and Filters

### DISCUSSION THREAD

**Discussion by aquasarus** [aquasarus/Starfield-Interactive-Smart-Slate#10] [Announcements]
v1.2 is out now! As the most requested feature, this update adds basic Outpost tracking to the app. I've also added filters to help sort through the hundreds of celestial bodies much more easily!

## Installation
Head to the [release page](https://github.com/aquasarus/Starfield-Interactive-Smart-Slate/releases) to download this. Simply extract and use! Your data is stored in a [separate folder](https://github.com/aquasarus/Starfield-Interactive-Smart-Slate/wiki/User-Data), and so will not be wiped.

#### Outpost Tracking
Just like Fauna/Flora, you can add pictures and notes to your outposts!

![Gif demo-ing outpost tracking](https://raw.githubusercontent.com/aquasarus/Starfield-Interactive-Smart-Slate/main/Gifs/v1.2_track_outposts.gif)

#### Filters
Use the Lifeform and Outpost filters to help find planets/moons you're interested in.

<img src="https://raw.githubusercontent.com/aquasarus/Starfield-Interactive-Smart-Slate/main/Gifs/v1.2_filters.gif" width=30% height=30%>

## Stats
This update cost **23 hours** of my free time. The most rewarding part of this is knowing that someone is out there enjoying the app! Those of you who allow me to collect anonymous usage data have collectively logged the following actions:

_Total Solar Systems Discovered: **260**
Total Lifeforms Discovered: **373**
Total Pictures Taken: **161**_

## Future Plans
Going forward, I'd like to start shipping smaller updates more frequently. This will allow me to deliver bug fixes to you guys much faster, since a lot of you are helping report them! Of course, it's always optional to download a newer version. Also, NO PROMISES :D, but here's a sneak peek of what I want to work on next:
- Searching for multiple resources simultaneously.
- Extending current pictures/notes functionality to planets and moons.
- The beginning of localization (supporting languages other than English).
- Integrating the "farmable" attribute of lifeforms into the app.

## Credits
Credit where credit is due. Some of you have directly or indirectly contributed to the development of **v1.2.0**. A heartfelt thank you is in order!

- aldhari
- DeHades
- laurac8r

**Comment by DeHades666** [comment]
Hey, great job on the update =) Really appreciate your hard work. As it stands the app is fantastic, the future plans making it even more invaluable. I can see it becoming a "must-have" for all budding explorers =)

**Reply by aquasarus** [reply]
Thanks. I can't overstate how much it means to me that you're enjoying this app. Although our true endgame is for the app to be rendered obsolete via official Bethesda additions, it would also be amazing to see it grow into a top utility for explorers. If that's the case, we will all play a part in it :).
